tests/integ/test_deepcopy.py

```diff
@@ -401,31 +401,15 @@ def test_deepcopy_no_duplicate(session, generator):
     copied_plan = copy.deepcopy(final_df._plan)
     check_copied_plan(copied_plan, final_df._plan)
 
-    # we will traverse the plan to assert that the tuple (plan._id, type(plan)) have a unique id(plan)
-    # note that two nodes with same plan._id can have different id(plan) since SnowflakePlan inherits plan._id
-    # from its source selectable.
-    #   plan._id: calculated by hashing the query sql and query params of plan
-    #   type(plan): the type of the plan
-    #   id(plan): the memory address of the plan object
-    # If the same plan._id has multiple id(plan), it means the deepcopy is duplicating source nodes
-
     def traverse_plan(plan, plan_id_map):
-        plan_id = plan._id
-        plan_type = type(plan)
```

Comment:
@sfc-gh-aalam i am not sure what are we trying to test here, if two nodes have the same query and same type, there should still be two nodes after deepcopy, because they are two different nodes.